tomviz/DataTransformMenu.cxx

```diff
@@ -71,8 +71,9 @@ void DataTransformMenu::buildTransforms()
   auto fftAbsLogAction = menu->addAction("FFT (abs log)");
   auto gradientMagnitudeSobelAction = menu->addAction("Gradient Magnitude");
   auto unsharpMaskAction = menu->addAction("Unsharp Mask");
-  auto laplaceFilterAction = menu->addAction("Laplace Filter");
-  auto gaussianFilterAction = menu->addAction("Gaussian Filter");
+  auto laplaceFilterAction = menu->addAction("Laplace Sharpen");
+  auto gaussianFilterAction = menu->addAction("Gaussian Blur");
```

Comment:
We should update the label in the JSON file as well.

Reply:
I mean update the label [here](https://github.com/OpenChemistry/tomviz/blob/master/tomviz/python/GaussianFilter.json#L3)